cfgov/unprocessed/js/modules/util/dom-manipulators.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var domTraverse = require( './dom-traverse' );
+
+/**
+ * Shortcut for creating new dom elements
+ * @param   {string} tag     The html elem to create
+ * @param   {object} options The options for building the elem
+ * @returns {HTMLNode}       The created elem
+ */
+function create( tag, options ) {
+  var elem = document.createElement( tag );
```

Comment:
So I think we could have separate modules for each and then have `dom-traverse` just require those files. The benefit is smaller files sizes for the particular module and you don't end up including a lot of JS that you don't use.